PreviewPlayerView.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  PreviewPlayerView.swift
+//  Recast
+//
+//  Created by Jack Thompson on 9/19/18.
+//  Copyright © 2018 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class PreviewPlayerView: UIView {
+
+    // MARK: - Variables
+    var upNextView: UIView!
+    private var upNextLabel: UILabel!
```

Comment:
No need to make these private, let's leave these public until we know for sure how it'll interact with our player and view controller.